src/Routes/ImageManager/Images.js

```diff
@@ -60,6 +61,10 @@ const Images = () => {
     });
   };
 
+  const retryImageBuild = (id) => {
+    retryImage(id);
```

Comment:
I need some ideas on how to handle the callback of this endpoint and show error / success messages to the user and update the table.

if 201 -> success message and update table, and start pooling the status like we do on image creation 
if not -> error